conf/configuration.go

```diff
@@ -131,6 +133,9 @@ func LoadConfig() (Config, error) {
 	if val, ok = os.LookupEnv("PINNER_LOG_LEVEL"); ok {
 		cfg.LogLevel = val
 	}
+	if val, ok = os.LookupEnv("PINNER_HOURS_BETWEEN_SCANS"); ok {
```

Comment:
I can't say why but I feel like hours is not something I'd expect when configuring a sleep.
I'd assume something like `PINNER_SLEEP_BETWEEN_SCANS` which by default accepts seconds and optionally allows for passing strings with suffixes like `24h`.

Reply:
agreed, I would prioritize consistency with other services here. Seconds is a much more common time interval.

Reply:
Added. It defaults to seconds when no suffix is given but also allows for time format such as `2h23m34s`.